manager/machine.go

```diff
@@ -60,6 +61,9 @@ func getClockSpeed(procInfo []byte) (uint64, error) {
 		return maxFreq, nil
 	}
 	// Fall back to /proc/cpuinfo
+	if strings.Contains(runtime.GOARCH, "ppc64") {
```

Comment:
I would avoid runtime and just check the speed if the x86 test fails. It will make testing it easier with a fake procinfo file.